README.md

```diff
@@ -27,7 +31,9 @@ symbolic link from the Mark
 
 or
 
-* A built and installed current nightly MarkLogic package.  If you've used this method, remove or rename the entity-services directory at `src/Modules/MarkLogic/entity-services`
+* A built and installed current nightly MarkLogic package.  If you've used this
+  method, remove or rename the entity-services directory at
+  `src/Modules/MarkLogic/entity-services`
 
 
 For either method of server development, you'll need to symbolically link
```

Comment:
Well, anyone who wants to do real-time changes to entity services itself will indeed have to.
Users of the library do not.
This message is for people who want to contribute to the library itself, but you're right, if contributions are to examples then that's not required.